### PR TITLE
[build] Disable EV build

### PR DESCRIPTION
**Pull request by poinwater**
# Description

Disabled Embedding Variables build in bazel build file due to the `ptr_util.h` header is removed from Tensorflow 2.13.
The EV is not in used, so it's safe to remove it.

Related file used `ptr_utl.h`: [ev_op_helpers.h](https://github.com/tensorflow/recommenders-addons/blob/7f20412d36d51fe95011931a6b0825c8d40ae093/tensorflow_recommenders_addons/embedding_variable/core/kernels/ev_op_helpers.h#L21)

Related commit: [960d53e](https://github.com/tensorflow/tensorflow/commit/960d53e1f74a0c1169a45cfb7173837c07bbb63a)

Brief Description of the PR: Disable EV build

Fixes # (issue)

## Type of change

- [ ] Bug fix
- [x] Build
- [ ] New Tutorial
- [ ] Updated or additional documentation
- [ ] Additional Testing
- [ ] New Feature

# Checklist:

- [x] I've properly [formatted my code according to the guidelines](https://github.com/tensorflow/recommenders-addons/blob/master/CONTRIBUTING.md#coding-style)
    - [x] By running yapf
    - [x] By running clang-format
- [ ] This PR addresses an already submitted issue for TensorFlow Recommenders-Addons
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

# How Has This Been Tested?

If you're adding a bugfix or new feature please describe the tests that you ran to verify your changes:
*

### CODE REVIEW

File: tensorflow_recommenders_addons/BUILD [hunk 1]
```diff
@@ -15,12 +15,13 @@ config_setting(
...
 )
 
+# Disable EV due to a required header file is removed from Tensorflow 2.13
 py_library(
     name = "tensorflow_recommenders_addons",
     srcs = glob(["**/*.py"]),
     deps = [
         "//tensorflow_recommenders_addons/dynamic_embedding/python/ops:dynamic_embedding",
-        "//tensorflow_recommenders_addons/embedding_variable/python:embedding_variable",
+        # "//tensorflow_recommenders_addons/embedding_variable/python:embedding_variable",
```
Comment (rhdong) [inline tensorflow_recommenders_addons/BUILD:24]: It's recommended that all of the comments be changed to remove directly, we can revert is needed in the future.

File: tensorflow_recommenders_addons/embedding_variable/python/ops/embedding_variable_ops_test.py [hunk 2]
```diff
@@ -1,158 +1,156 @@
...
-#
-# Author: Chen Ding <cnady.dc@alibaba-inc.com>
-# Created: 2018/03/26
-# Description:
-# ==============================================================================
-"""Tests for tensorflow_recommenders_addons.python.ops.embedding_variable."""
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
+# # Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```
Comment (rhdong) [inline tensorflow_recommenders_addons/embedding_variable/python/ops/embedding_variable_ops_test.py:1]: Delete the file directly.

File: tensorflow_recommenders_addons/BUILD (hunk 1 above)
Comment (poinwater) [reply tensorflow_recommenders_addons/BUILD]: Removed

**Review by rhdong — APPROVED** [review]
LGTM

**Comment by rhdong** [comment]
Hello @poinwater, the CI failure is caused by disk no space. Your code is perfect, and I can help merge it if no more modify from you. Thank you!

**Comment by poinwater** [comment]
@rhdong , sure, please merge my code, thank you.

**Pull request merged**